doc/api/events.md

```diff
@@ -1085,6 +1085,54 @@ newListeners[0]();
 emitter.emit('log');
 ```
 
+### `emitter.use(eventName, listener)`
```

Comment:
A lot of different things can extend `EventEmitter` class. The name `use()` is too generic and might  also collide with existing userspace code, I think `useEventListener()` (or `addDisposableEventListener()`, `onUse()`, `scopedOn()`, etc., it's quite bikesheddable...) would be better.